data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2830,7 +2830,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "May you shut your ugly mouth and hurry up and die."
+            message= _ "May you shut your ugly mouth and hurry up and die!"
```

Comment:
I prefer the original, it's a quiet contrast to the screaming rant from Malifor.

Reply:
Debatable. If someone agrees, I'l change it.

Reply:
Reading on its own, the exclamation mark makes sense. But reading it in the context of Malifor's ranting - I also prefer the quiet, almost bored response from Tallin. I find _this_ aspect funny, in contrast with the poor fire puns.

Reply:
Please leave the original too.

Reply:
OK.